miqa/core/models/scan.py

```diff
@@ -40,7 +40,7 @@ class Meta:
         ordering = ['scan_id']
 
     id = models.UUIDField(primary_key=True, default=uuid4, editable=False)
-    experiment = models.ForeignKey('Experiment', on_delete=models.CASCADE)
+    experiment = models.ForeignKey('Experiment', related_name='scans', on_delete=models.CASCADE)
```

Comment:
I believe these changes technically require a DB migration even though they don't (to my knowledge) modify the database.